src/chains/eos_2/eosTransaction.ts

```diff
@@ -345,9 +362,13 @@ export class EosTransaction implements Transaction {
 
   /** An array of authorizations (e.g. account/permission) that do not have an attached signature
    *  Retuns null if no signatures are missing */
-  public get missingSignatures(): EosAuthorization[] {
+  public get missingSignatures(): EosAuthorizationPerm[] {
```

Comment:
This getter function doesn't work as expected anymore.

I believe we actually want to confirm that the threshold is met for each action. Something like:
```
const missing = this._requiredAuthorizations?.filter(auth => {
      return !this.checkAuthSigned(auth.requiredAuthorization);
    })
```

Reply:
done